src/app/backup/list/restore/component.ts

```diff
@@ -127,7 +127,7 @@ export class RestoreListComponent implements OnInit, OnDestroy {
         )
       )
       .subscribe(_ => {
-        this._notificationService.success(`Successfully deleted restore object ${restore.name}`);
+        this._notificationService.success(`Deleted restore object ${restore.name}`);
```

Comment:
For e.g. automatic backups we use `Deleted the ${backup.name} automatic backup` (so basically Action-Name-Resource)
and here we use `Deleted restore object ${restore.name}` (Action-Resource-Name)

I guess we should think about one order in general, shouldn't we?
[Haven't checked on all yet, but was the first one where I noticed it.]

Reply:
cc @cschieder

Reply:
IMO the first form is better.

Reply:
Iirc we use action-name-resource basically everywhere, like kristin said. another example: "Delete silly-clustername cluster" Imo it reads well.